docs/getting_started.md

```diff
@@ -12,7 +12,7 @@ The required versions of MMCV, MMDetection and MMSegmentation for different vers
 
 | MMDetection3D version | MMDetection version | MMSegmentation version |    MMCV version     |
 |:-------------------:|:-------------------:|:-------------------:|:-------------------:|
-| master              | mmdet>=2.10.0, <=2.11.0| mmseg>=0.13.0 | mmcv-full>=1.3.1, <=1.4|
+| master              | mmdet>=2.12.0          | mmseg>=0.13.0 | mmcv-full>=1.2.4, <=1.4|
```

Comment:
should >=1.3.1?

Reply:
sorry，we should >=1.3.2, because mmdet need mmcv-full version >= 1.3.2